debian/homestead.init.d

```diff
@@ -160,6 +160,7 @@ get_daemon_args()
 
 
         [ "$sas_use_signaling_interface" != "Y" ] || sas_signaling_if_arg="--sas-use-signaling-interface"
+        [ "$support_shared_ifc" == "N" ] || support_shared_ifcs_arg="--support-shared-ifcs"
```

Comment:
Should we do != Y to match the `sas_user_signaling_interface` here? I think it's probably better if 'n' and 'no' don't match, rather than 'yes' matching?

Alternatively, should we have some common code which checks whether a value is truthy?